lib/api/1.1/northbound/nodes.js

```diff
@@ -426,6 +426,19 @@ function nodesRouterFactory (
         return nodeApiService.delActiveWorkflowById(req.params.identifier);
     }));
 
+    router.get('/nodes/:identifier/tags', rest(function(req) {
+        return nodeApiService.getTagsById(req.params.identifier);
+    }));
+
+    router.patch('/nodes/:identifier/tags', rest(function (req) {
+        if(req.body.tags) {
+            return nodeApiService.addTagsById(req.params.identifier, req.body.tags);
+        }
+    }));
+
+    router.delete('/nodes/:identifier/tags/:name', rest(function(req) {
```

Comment:
need annotations, v1.1 api apidoc, and python-client is based on router's annotations.